ncbi-virus-url

```diff
@@ -20,9 +21,13 @@ def parse_args():
         help="NCBI Taxon ID. Visit NCBI virus at " +
              "https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/find-data/virus " +
              "to search for supported taxon IDs.")
+    parser.add_argument("--filters", required=False, nargs="*",
+        help="Filter criteria to add as `fq` param values. " +
+             "Apply filters via the NCBI Virus UI and observe the network " +
+             "activity to find the desired filter string.")
```

Comment:
In our latest meeting with WA DOH, people were concerned whether the clean data that Nextstrain hosts at data.nextstrain.org/files might exclude certain records from NCBI. 

Allowing customizable filters here might add to that concern, but there's already so many other points in ingest pipelines that might filter out records. I think we just need to be better about documenting these filters that are used to generate the clean data.